### PR TITLE
Phase 2: Copilot Auto-Fix integration

### PR DESCRIPTION
**Pull request by mrunalhirve128**
## Summary
Phase 2 of the auto-triage system: Copilot Auto-Fix integration with AI-powered daily reports.

## How Copilot Auto-Fix Works

When a new GitHub issue is created, our auto-triage system sends the issue title and body to an LLM (GitHub Models/Azure OpenAI) which evaluates whether the issue is suitable for Copilot to fix autonomously. The LLM follows criteria defined in `prompts.yaml` - it returns `true` for well-scoped issues like typo fixes, documentation updates, simple bug fixes, and code formatting; it returns `false` for complex issues requiring architectural decisions, security-sensitive code, or multi-file coordination.

If the LLM determines an issue is Copilot-fixable, our code automatically assigns the issue to `copilot-swe-agent` via the GitHub API. Once assigned, GitHub's built-in Copilot coding agent takes over - it reads the issue, analyzes the codebase, implements the fix, and creates a draft PR for human review.

**Our code vs GitHub's Copilot:** Our autoTriage code only handles the *decision* (is this Copilot-fixable?) and the *assignment* (assign to Copilot). The actual code fix and PR creation is done entirely by GitHub's Copilot infrastructure.

**Verified:** Tested with issue #237 (indentation fix) - Copilot successfully created PR #238 with the correct fix within minutes of being assigned.

### Copilot Auto-Fix Integration
- Created `CopilotService` to invoke GitHub Copilot coding agent
- Added `is_copilot_enabled()` to check if repo supports Copilot
- Added `assign_to_copilot()` to assign issues via REST API
- Added `get_fix_instructions()` to generate custom instructions for Copilot
- Integrated CopilotService into `intake_service._apply_triage_changes`
- When issue is copilot-fixable, invokes Copilot coding agent automatically
- Updated triage comment to indicate Copilot assignment
- **Enabled Copilot auto-fix by default** in config parser

### AI Summary for Daily Reports
- Added AI-generated summary section to daily issue reports
- New prompts in `prompts.yaml`: `daily_report_summary_system` and `daily_report_summary_user`
- Added `generate_ai_summary()` method to `DailyReportService`
- Summary includes: overview, SLA status, priority breakdown, and recommendations

### CLI Reorganization
- Moved CLI scripts to `cli/` folder for better organization:
  - `daily_report.py` -> `cli/daily_report.py`
  - `escalation_check.py` -> `cli/escalation_check.py`
  - `triage_issue.py` -> `cli/triage_issue.py`
- Updated all GitHub workflow files to use new paths

### Testing
- Added comprehensive parameterized tests for `CopilotService` (64 tests)
  - Copilot detection in actors list (14 edge cases)
  - Malformed API responses (9 scenarios)
  - Error handling (7 scenarios: timeout, auth, network, etc.)
  - Invalid JSON responses (5 cases)
  - Assignment success/failure scenarios
  - Fix instructions generation with special characters
- Added AI summary tests to `DailyReportService` (6 tests)
- Added config tests for `copilot_enabled` default (4 tests)
- **Total: 111 tests passing**

### Other Fixes
- Fixed `team-members.json` login (mrunalhirve -> mrunalhirve128)
- Added logging to `github_service.assign_issue`
- Updated README with new features and folder structure

## Test Results
```
============================= 111 passed in 1.17s =============================
```

### CODE REVIEW

File: autoTriage/services/intake_service.py [hunk 1]
```diff
@@ -335,13 +339,63 @@ def _apply_triage_changes(
...
+                copilot_service = CopilotService()
+                
+                # Check if Copilot is enabled for this repo
+                if copilot_service.is_copilot_enabled(owner, repo):
+                    # Generate custom instructions from fix suggestions
+                    custom_instructions = copilot_service.get_fix_instructions(
+                        issue_title=classification.issue_url.split('/')[-1] if classification.issue_url else "",
+                        issue_body="",
+                        fix_suggestions=classification.fix_suggestions or []
+                    )
```
[the remark this answers is not in the document]
Comment (mrunalhirve128) [reply autoTriage/services/intake_service.py]: Fixed
